build_tools/get_tag.py

```diff
@@ -0,0 +1,25 @@
+from pathlib import Path
+import os
+
+# This file assumes that our tags are always in this format: vX.X.X.
+# In that case, we would only want to write X.X.X
+
+OUT_FILE = Path(__file__).parent / 'VERSION'
```

Comment:
I think this needs to be if we want it to be shipped with the package

```python
OUT_FILE = Path(__file__).parent / 'pmdarima' / 'VERSION'
```

Reply:
Right, but does it need to be shipped with the package? Currently, `pmdarima.__version__` contains all of the version info. That is still happening in this branch, but it just reads from `../VERSION`

Reply:
Looks like you fixed this!

Reply:
Yes it needs to be shipped since importing `pmdarima` will hit that line in the `__init__` where it tries to load the VERSION. I've included the version in the manifest, so it's bundled with the install.